config/routes.rb

```diff
@@ -178,6 +178,19 @@
       get 'management_information/download', to: 'management_information#download', as: :management_information_download
       get 'management_information/generate', to: 'management_information#generate', as: :management_information_generate
     end
+
+  end
+
+  namespace :provider_management do
+    root to: 'provider_management/providers#index'
+    get 'external_users/find', to: 'external_users#find'
+    post 'external_users/find', to: 'external_users#search'
```

Comment:
are these two routes used - they don't appear to be.

Reply:
Yes, they are called from the provider_management index view... ```/provider_management/external_users/find``` it's the find provider by email page

Reply:
nor even for superadmin?!

Reply:
For superadmin too... try logging in to live and click the 'Find provider by email` link and see where you end up! :)

Reply:
oh yeah. had my code searching limited to specs. urrgh!